base_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './nameable'
+
+class BaseDecorator
```

Comment:
- [ ] Please make that the `BaseDecorator` class inherits from the `Nameable` class :ok_hand: . Because one of the requirements says this :point_down: 
![Screenshot from 2023-03-07 16-53-50](https://user-images.githubusercontent.com/89228743/223492914-54ce31d5-b383-4f74-9b06-e5dd2db29156.png)